mozilla-release/config/version_win.pl

```diff
@@ -274,11 +274,11 @@ sub getNextEntry
 
 my $versionlevel=0;
 my $insideversion=0;
-if (open(RCINCLUDE, "<$rcinclude")) 
+if (open(RCINCLUDE, "<$rcinclude"))
 {
 	print RCFILE "// From included resource $rcinclude\n";
 #	my $mstring="";
-	while (<RCINCLUDE>) 
+	while (<RCINCLUDE>)
```

Comment:
Do agree that trailing spaces suck, but this file is very old file (according to gecko-dev on github it was not changed in last 3 years), in newer ones there should be no trailing spaces. 

One thing that we can to is to remove all trailing spaces from mozilla-central and submit a patch to bugzilla. 

If we want to do that we can keep those changes now.